src/components/Compare/Compare.style.tsx

```diff
@@ -488,3 +489,35 @@ export const StateName = styled.div`
   font-size: 0.9rem;
   line-height: 1.2;
 `;
+
+export const HeaderContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0 0 1.5rem 0;
+  .MuiToggleButtonGroup-root {
+    margin-top: 0.5rem;
+  }
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    justify-content: flex-start;
+  }
+
+  @media (min-width: 375px) {
+    .MuiToggleButtonGroup-root {
+      margin: 0;
+    }
+  }
+`;
+
+export const CompareHeader = styled.h2`
```

Comment:
instead of re-writing the header, lets import SectionHeader from SharedComponents.style and iterate on that (so they can stay relatively in sync), so

`const CompareHeader = styled(SectionHeader)`
// styles that are specific to the compare header
``